smithed_libraries/packs/prevent-aggression/beet.yaml

```diff
@@ -0,0 +1,17 @@
+extend: "@smithed_libraries/common.yaml"
+
+id: smithed.prevent_aggression
+name: Smithed Prevent Aggression
+description: Stop mobs from targetting your custom mobs
+version: '0.1.0'
+
+data_pack:
+  name: Smithed Prevent Aggression
+  load: .
+
+pipeline:
+  - mecha
+ 
+overrides:
+  - > 
+    meta.smithed.versioning = {"scheduled_paths": "impl/technical/10_tick"}
```

Comment:
The `>` is only needed when the yaml string spans multiple lines and should be folded back into a single line.

```yml
overrides:
  - meta.smithed.versioning = {"scheduled_paths": "impl/technical/10_tick"}
```

Reply:
Yea, i suppose I wanted to make sure the override was copied w/o any quoting issues properly
ur code snippet isn't valid YAML according to VSCode

Reply:
My bad yeah you'd still need single quotes

```yml
overrides:
  - 'meta.smithed.versioning = {"scheduled_paths": "impl/technical/10_tick"}'
```